<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReverseEdit</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="TabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ConfigTab">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="SourceVideoList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="SourceButtonLayout">
          <item>
           <widget class="QPushButton" name="AddVideoButton">
            <property name="text">
             <string>Add Video</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RemoveVideoButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove Video</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SetEditVideoButton">
            <property name="text">
             <string>Set Edit Video</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="OptionsBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>125</height>
           </size>
          </property>
          <property name="title">
           <string>Options</string>
          </property>
          <widget class="QSpinBox" name="EditFrameSkipSpinbox">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>20</y>
             <width>91</width>
             <height>26</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="SourceFrameSkipSpinbox">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>50</y>
             <width>91</width>
             <height>26</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="FrameScanRangeSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>80</y>
             <width>91</width>
             <height>26</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
          <widget class="QCheckBox" name="IncrementalCheckbox">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>50</y>
             <width>121</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Incremental</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="IncrementalRangeCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>80</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Incremental Range</string>
           </property>
          </widget>
          <widget class="QLabel" name="FrameScanRangeLabel">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>80</y>
             <width>131</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Frame Scan Range:</string>
           </property>
          </widget>
          <widget class="QLabel" name="SourceFrameSkipLabel">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>50</y>
             <width>131</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Source Frame Skip:</string>
           </property>
          </widget>
          <widget class="QLabel" name="EditFrameSkipLabel">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>20</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Edit Frame Skip:</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="PerceptualHashButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>151</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Perceptual Hashing</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="AverageHashButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>151</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Average Hashing</string>
           </property>
          </widget>
          <widget class="QPushButton" name="StartButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>500</x>
             <y>80</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PreviewTab">
       <attribute name="title">
        <string>Preview</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="PreviewLayout">
          <item>
           <widget class="QGroupBox" name="PreviewGroupBox">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Preview</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="MediaProgressBar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="MediaPlayButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ExportTab">
       <attribute name="title">
        <string>Export</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionOpenWorkspace"/>
    <addaction name="ActionSaveWorkspace"/>
    <addaction name="ActionExit"/>
   </widget>
   <addaction name="MenuFile"/>
  </widget>
  <action name="ActionOpenWorkspace">
   <property name="text">
    <string>Open Workspace</string>
   </property>
  </action>
  <action name="ActionSaveWorkspace">
   <property name="text">
    <string>Save Workspace</string>
   </property>
  </action>
  <action name="ActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
